src/containers/Register/AddProduct/index.js

```diff
@@ -142,19 +148,15 @@ const AddProduct = ({
               placeholder="Selecione o produto"
               optionFilterProp="children"
               filterOption={(input, option) =>
-                option.props.children
-                  .toLowerCase()
-                  .indexOf(input.toLowerCase()) >= 0
+                option.props.children.toLowerCase().indexOf(input.toLowerCase()) >= 0
               }
               name="mark"
               value={mark}
-              style={{ width: "100%" }}
-              onChange={value => handleOnChange({ target: { value, name: 'mark' } })}
-              onSearch={mark => getAllMarca(mark)}
+              style={{ width: '100%' }}
+              onChange={(value) => handleOnChange({ target: { value, name: 'mark' } })}
+              onSearch={(mark) => getAllMarca(mark)}
             >
-              {marksList && marksList.map(item => (
-                <Option value={item.mark}>{item.mark}</Option>
-              ))}
+              {marksList && marksList.map((item) => <Option value={item.mark}>{item.mark}</Option>)}
```

Comment:
Precisamos adicionar a Key do react para os primeiro elementos 
```suggestion
              {marksList && marksList.map((item) => <Option key={item.id} value={item.mark}>{item.mark}</Option>)}
```